.github/workflows/windows.yml

```diff
@@ -112,11 +112,10 @@ jobs:
 
       - name: Build
         run: |
-          # TODO: remove the build-dir and install-prefix flags once dev.py
-          # supplies meson with absolute paths. This is due to a pathlib bug
-          # on cp39-win.
-          python dev.py --build-dir=$pwd\build --install-prefix=$pwd\build-install build -j 2 --win-cp-openblas
-          meson install -C build
+          python dev.py build -j 2 --win-cp-openblas
+          # Necessary because GitHub Actions checks out the repo to D:\ while OpenBLAS
+          # got installed to C:\ higher up. The copying with `--win-cp-openblas` fails
+          # when things are split over drives.
```

Comment:
What's the ultimate reason copying isn't possible between different drives?

Reply:
Windows design, as far as I can tell. Unix-like systems have a single filesystem starting from `/` with mount points. Windows drives are really separate, no common `/`. Windows Explorer and GUI tools know how to deal with that, Python and Bash CLI tools don't.